README.md

```diff
@@ -652,344 +652,319 @@ Let’s go!
+      "gas_consumed" : "7565310",
+      "notifications" : [
+         {
+            "contract" : "0x081480331bcd3183cd6c4d0c035717bba1b4daae",
+            "state" : {
+               "value" : [
+                  {
+                     "value" : "dHJhbnNmZXI=",
+                     "type" : "ByteString"
+                  },
+                  {
+                     "type" : "ByteString",
+                     "value" : "E9RIbx59YF1EGCw9f0E19Ms6o/A="
+                  },
+                  {
+                     "type" : "ByteString",
+                     "value" : "50l6vFaauRKm8hPVkr3Aw2CeHQs="
+                  },
+                  {
+                     "type" : "Integer",
+                     "value" : "500000000"
+                  }
+               ],
+               "type" : "Array"
+            }
+         }
+      ]
+   }
 }
 ```
 Note, that `stack` field contains `1`, which means that token was successfully transferred.
-Let's now check the balance of `AKkkumHbBipZ46UMZJoFynJMXzSRnBvKcs` account to ensure that the amount of token on that account = 5:
+Let's now check the balance of `NgzuJWWGVEwFGsRrgzj8knswEYRJrTe7sm` account to ensure that the amount of token on that account = 5:
 ```
-./bin/neo-go contract invokefunction -e http://localhost:20331 -w my_wallet.json -g 0.00001 f84d6a337fbc3d3a201d41da99e86b479e7a2554 balanceOf AKkkumHbBipZ46UMZJoFynJMXzSRnBvKcs
+$ ./bin/neo-go contract invokefunction -r http://localhost:20331 -w my_wallet.json 081480331bcd3183cd6c4d0c035717bba1b4daae BalanceOf NgzuJWWGVEwFGsRrgzj8knswEYRJrTe7sm
 ```
 The `getapplicationlog` RPC-call for this transaction tells us the following:
 ```
 {
+   "id" : 1,
    "result" : {
-      "txid" : "0x172c5074646ce043095e612d31e5c5c3d00c7c8a4e8c01873cc732692d5152f5",
-      "executions" : [
+      "gas_consumed" : "4228110",
+      "txid" : "0x41e24302e393012b2a23859367adf50c931d66ded48926d3e2b9f8b2acc00f86",
+      "stack" : [
          {
-            "stack" : [
-               {
-                  "value" : "05",
-                  "type" : "ByteArray"
-               }
-            ],
-            "gas_consumed" : "0.209",
-            "notifications" : [],
-            "vmstate" : "HALT",
-            "trigger" : "Application",
-            "contract" : "0xbaaafb6be440d1de3d298ba556ad23aa0209ef2f"
+            "value" : "500000000",
+            "type" : "Integer"
          }
-      ]
+      ],
+      "trigger" : "Application",
+      "vmstate" : "HALT",
+      "notifications" : []
    },
-   "id" : 1,
    "jsonrpc" : "2.0"
 }
 ```
-Here we are! There are exactly 5 tokens at the `stack` field. You can also ensure that these 5 tokens were debited from `AK2nJJpJr6o664CWJKi1QRXjqeic2zRp8y` account by using `balanceOf` method.
+Here we are! There are exactly 5 tokens at the `stack` field. You can also ensure that these 5 tokens were debited from `NMipL5VsNoLUBUJKPKLhxaEbPQVCZnyJyB` account by using `BalanceOf` method.
```

Comment:
It just occured to me that this actually breaks NEP5. It must be `balanceOf` to conform the standard.

Reply:
And the other methods have the same issue.